test/basic-querying.test.js

```diff
@@ -591,6 +592,12 @@ describe('basic-querying', function() {
             });
         });
 
+      it('should sanitize invalid usage of like', async () => {
```

Comment:
Why should an invalid usage of find with '[' return an actual result?

Reply:
I am confused because another test below returns `empty`. This is what I would expect here as well.

e.g.

![image](https://user-images.githubusercontent.com/6864736/66845125-7cf53900-ef3d-11e9-9334-740ca8852fba.png)

Reply:
These tests were made under the context described in my reponse to @bajtos comment.  I approached it by choosing to interpret an invalid RegExp string as a literal (escaped) one, which seraches for the actual characters provided.

The first test returns the result because of the regex search on new **tag** field which matches the `[singer]` content, while the `ilike` is on the name field which does not match.

If you guys decide that you just simply want to return the error as discussed with @bajtos then I will have to change these tests.